===================================================================
RCS file: /cvs/gnome/nautilus/nautilus-folder-handler.desktop.in,v
retrieving revision 1.6
retrieving revision 1.6.2.1
diff -u -r1.6 -r1.6.2.1
--- nautilus-folder-handler.desktop.in	2006/03/19 14:23:39	1.6
+++ nautilus/nautilus-folder-handler.desktop.in	2006/05/17 17:07:13	1.6.2.1
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Encoding=UTF-8
-Name=Open Folder
+_Name=Open Folder
 TryExec=nautilus 
 Exec=nautilus --no-desktop %U
 NoDisplay=true
===================================================================
RCS file: /cvs/gnome/nautilus/po/POTFILES.in,v
retrieving revision 1.166
retrieving revision 1.166.2.1
diff -u -r1.166 -r1.166.2.1
--- POTFILES.in	2006/03/18 07:13:49	1.166
+++ po/POTFILES.in	2006/05/17 17:07:14	1.166.2.1
@@ -36,6 +36,7 @@
 libnautilus-private/nautilus-undo-signal-handlers.c
 nautilus-computer.desktop.in
 nautilus-file-management-properties.desktop.in
+nautilus-folder-handler.desktop.in
 nautilus-home.desktop.in
 nautilus.desktop.in
 src/Nautilus_shell.server.in
